src/components/Header/index.tsx

```diff
@@ -243,8 +244,8 @@ export default ({ search = true }: { search?: boolean }) => {
               <Search />
             </div>
             <div className="header__testnet__panel">
-              <div className="header__testnet__flag">TESTNET</div>
-              <div className="header__testnet__tip">Mainnet is comming</div>
+              <div className="header__testnet__flag">{i18n.t('navbar.button_text')}</div>
```

Comment:
`TESTNET` is testnet flag and is not a button.

`button_text` is too general and maybe we should make a more meaningful name